src/services/transactionHistory.ts

```diff
@@ -124,7 +137,15 @@ const askTransactionSqlQuery = `
           JOIN tx source_tx 
             ON source_tx_out.tx_id = source_tx.id
           where inadd_tx.hash = tx.hash) as "inAddrValPairs"
-       , (select json_agg(("address", "value") order by "index" asc)  as outAddrValPairs
+       , (select json_agg((
+                    "address", 
+                    "value",
+                    (select json_agg(ROW("policy", "name", "quantity"))
```

Comment:
usually we use `encode(...., 'hex')` to get rid of the `\\x` prefix on the results.